src/routes/alerts/guards/tenderly-signature.guard.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  CanActivate,
+  ExecutionContext,
+  Inject,
+  Injectable,
+} from '@nestjs/common';
+import { Request } from 'express';
+import * as crypto from 'crypto';
+import { IConfigurationService } from '@/config/configuration.service.interface';
+
+@Injectable()
+export class TenderlySignatureGuard implements CanActivate {
+  private static readonly SIGNATURE_HEADER = 'x-tenderly-signature';
+  private static readonly TIMESTAMP_HEADER = 'date';
+
+  private readonly signingKey: string;
+
+  constructor(
+    @Inject(IConfigurationService)
+    private readonly configurationService: IConfigurationService,
+  ) {
+    // Dashboard key for specific webhook
```

Comment:
I move this description to the environment variable itself (in `.env.sample` for example) as it's the entry point of contact for the environment configuration (for users deploying the service).

Reply:
As we [already have a description in the .env.sample](https://github.com/safe-global/safe-client-gateway/blob/40b5b7e3bc64689c32778bf570fe515d9ba972a7/.env.sample#L17), I removed this comment in 40b5b7e3bc64689c32778bf570fe515d9ba972a7.